### PR TITLE
fix(apigatewayv2-authorizers): cannot add multiple user pool authorizers without specifying name 

### PR DESCRIPTION
**Pull request by peterwoodworth**
fixes: #13711

BREAKING CHANGE: Users now have to specify `HttpUserPoolAuthorizer` and `HttpJwtAuthorizer` prop `authorizerName`

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by peterwoodworth** [comment]
[HttpLambdaAuthorizer](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-apigatewayv2-authorizers.HttpLambdaAuthorizerProps.html) already requires the `authorizerName` prop and doesn't check for tokens like the other Authorizers do. Should we update this construct to check for tokens as well?

**Review by nija-at — CHANGES_REQUESTED** [review]
Hey @peterwoodworth - can you summarize the bug and your fix in the PR description? 
It would be helpful to review this. The issue linked has a few things going on that is hard to follow.

Once done, dismiss my review so it comes back to my plate.

**Comment by nija-at** [comment]
Closing this since there's no response. Please reopen when ready.
